tests/CMakeLists.txt

```diff
@@ -162,6 +163,10 @@ set_property(TEST nodeos_repeat_transaction_lr_test PROPERTY LABELS long_running
 
 add_test(NAME cli_test COMMAND tests/cli_test.py WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 
+add_test(NAME plugin_http_api_test COMMAND tests/plugin_http_api_test.py -v --clean-run --dump-error-detail WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
+set_tests_properties(plugin_http_api_test PROPERTIES TIMEOUT 3000)
```

Comment:
You mentioned in an IM that your test takes about 20 seconds to run.  
A timeout of 3,000 seems excessive for a test which takes 20 seconds to run. When choosing a timeout, consider that you have exceptional hardware and it might take longer for a community contributor to run the same test.

Reply:
changed to 40 seconds,